{"name": "test_login_sucess", "status": "broken", "statusDetails": {"message": "TypeError: get_logout_text() takes 1 positional argument but 2 were given", "trace": "self = <testcase.test_login.TesttLogin object at 0x0000025512C4BB00>\nhandle = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"99c56ec3ce12a1074dce2fd17f0fee51\")>, <pages.login_page.LoginPage object at 0x0000025512C91860>)\n\n    def test_login_sucess(self,handle):\n    \n        handle[1].login(success_data['username'],success_data['password'])\n        handle[1].wait(5,'正在登录页面')\n>       assert handle[1].get_logout_text('获取登录成功后的退出文本内容') == success_data['msg']\nE       TypeError: get_logout_text() takes 1 positional argument but 2 were given\n\ntestcase\\test_login.py:13: TypeError"}, "attachments": [{"name": "log", "source": "875a3d42-2727-4f7f-a2b6-b9139799f0ba-attachment.txt", "type": "text/plain"}], "start": 1647791265299, "stop": 1647791270550, "uuid": "29d771a9-63e9-452c-ae88-f86ea4095cdf", "historyId": "b92c15650b11611da4ad3585a44c42c5", "testCaseId": "d7d1835804eed54393e4a5e3eec250a6", "fullName": "testcase.test_login.TesttLogin#test_login_sucess", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TesttLogin"}, {"name": "host", "value": "WIN-N5AGDL06G55"}, {"name": "thread", "value": "81480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}